meta-lxatac-bsp/recipes-kernel/linux/files/defconfig

```diff
@@ -3880,6 +3881,9 @@ CONFIG_DRM_CLIENT_SETUP=y
 CONFIG_DRM_FBDEV_EMULATION=y
 CONFIG_DRM_FBDEV_OVERALLOC=100
 CONFIG_DRM_FBDEV_LEAK_PHYS_SMEM=y
+# CONFIG_DRM_CLIENT_LOG is not set
+CONFIG_DRM_CLIENT_DEFAULT_FBDEV=y
+CONFIG_DRM_CLIENT_DEFAULT="fbdev"
```

Comment:
fbdev shouldn't be used at all, can we turn of the FBDEV emulation completely please?

Reply:
Sadly no. We use the framebuffer emulation in the [`tacd`](https://github.com/linux-automation/tacd) for historic reasons (the display we originally intended to use did not have a DRM driver, so we build the `tacd` around that).
I've made some previous attempts at switching the `tacd` to DRM, but never found the time to bring that to completion.